### DISCUSSION TITLE
Change the hostname

### DISCUSSION THREAD

**Discussion by sudokit** [AlexProgrammerDE/balena-minecraft-server#92] [Q&A] [answered]
How do i change the server hostname. I cant see a DEVICE_HOSTNAME Environment Variable within balenaCloud.

**Comment by AlexProgrammerDE** [comment, marked as answer]
> 
> 
> How do i change the server hostname. I cant see a DEVICE_HOSTNAME Environment Variable within balenaCloud.

Hi! You need to create the variable yourself. You find the place to do that under environment variables in the dashboard.
